static/styles/_base/_forms.scss

```diff
@@ -90,7 +90,24 @@ input[type="file"] {
 select {
   width: auto;
   max-width: 100%;
-  background: #fff;
+  -webkit-appearance: none;
```

Comment:
Is there a bourbon mixin to use for this rather then defining manually multiple times?